input/pagecontent/comparison.md

```diff
@@ -1,1206 +1,395 @@
+Compliance refers to a resource meeting all the requirements, constraints, and standards specified in a particular FHIR profile or an implementation guide. These requirements can include mandatory elements, cardinality constraints, data types, terminology bindings, and extensions. A resource may be compliant with multiple implementation guides. For example, in addition to AU Core profiles, a FHIR resource may be compliant with profiles published in project implementation guides, other national implementation guides, or international implementation guides.
```

Comment:
This text belongs under the next heading a maybe para 2 or 3. As in, the next heading after changes will say "Profile comparison" and there needs to be roughly an intro para, then a para on compliance, then the legend, then the table.

Reply:
1. confirmed this paragraph was moved
2. the order has been arranged
3. the introductory paragraph isn't introductory, it's a good content paragraph but the profile comparison section needs an introductory paragraph.
4. The paragraph "The table below provides a profile only comparison fr" can't stay where it is, it's in the middle of the page so makes no sense. This paragraph is a candidate for being reworked into the intro paragraph. Or it needs to be reworked in some other way and put further down the page.

Reply:
updated